versions/draft-3/SPEC.md

```diff
@@ -2152,7 +2180,7 @@ The `read_int()` function takes a file path which is expected to contain 1 line
 
 If the entire contents of the file can not be read for any reason, the calling task or workflow will be considered to have failed. 
 
-## String read_string(String|File)
+## String read_string(String|File|FileURI)
```

Comment:
There are more `read_x` functions which should also support `FileURI`

Reply:
@cjllanwarne Apologies, if I am missing something obvious, but I could not find any.  

No action.